ext/hal/st/stm32cube/CMakeLists.txt

```diff
@@ -113,6 +129,12 @@ if(CONFIG_SOC_SERIES_STM32L4X)
   zephyr_sources_ifdef(CONFIG_PWM stm32l4xx/drivers/src/stm32l4xx_hal_tim.c)
   zephyr_sources_ifdef(CONFIG_I2C stm32l4xx/drivers/src/stm32l4xx_ll_i2c.c)
   zephyr_sources_ifdef(CONFIG_SPI_STM32 stm32l4xx/drivers/src/stm32l4xx_ll_spi.c)
+  zephyr_sources_ifdef(CONFIG_USB_DC_STM32
+    stm32f4xx/drivers/src/stm32f4xx_ll_usb.c
+    stm32f4xx/drivers/src/stm32f4xx_hal_pcd.c
+    stm32f4xx/drivers/src/stm32f4xx_hal_pcd_ex.c
+    )
```

Comment:
Shouldn't these be stm32l4xx/drivers/...?

Reply:
Yes, I will fix it.